src/Homework4/Languages_special.py

```diff
@@ -0,0 +1,39 @@
+"""
+Языки
+Каждый из N школьников некоторой школы знает Mi языков.
+Определите, какие языки знают все школьники и языки,
+которые знает хотя бы один из школьников.
+"""
+
+
+def students(n, m1, m2, m3):
```

Comment:
unclear input. 
why do we need two solution? let only one be left

Reply:
delete file